manifests/daemon.pp

```diff
@@ -75,7 +75,7 @@
   $init_style         = $::prometheus::params::init_style,
   $service_ensure     = 'running',
   $service_enable     = true,
-  $manage_service     = true,
+  $manage_service     = $::prometheus::params::manage_service,
```

Comment:
The parameter should be taken from $::prometheus::manage_service ! Not taking default from params.pp